spec/examples/actions/projects.spec.js

```diff
@@ -38,4 +43,26 @@ describe('projectActions', () => {
       );
     });
   });
+
+  describe('changeCurrentProject', () => {
+    let projectKey;
+
+    beforeEach(() => {
+      store.dispatch(createProject());
+      projectKey = getCurrentProject(store.getState()).projectKey;
+      store.dispatch(toggleLibrary(projectKey, 'jquery'));
+      store.dispatch(createProject());
+      const secondProjectKey = getCurrentProject(store.getState()).projectKey;
+      store.dispatch(toggleLibrary(secondProjectKey, 'jquery'));
+      store.dispatch(changeCurrentProject(projectKey));
```

Comment:
@alexpelan what do you think of this sort of thing? Specifically, setting up a particular state for the store by firing off a bunch of actions to put it in that state? The main alternative on my mind would be directly specifying what the initial state should be (probably using an abstraction specifically built for tests)